google-cloud-bigquery-storage/src/test/resources/application.conf

```diff
@@ -1 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
 akka.http.server.preview.enable-http2 = on
```

Comment:
nit: Can we add new line here

Reply:
2 blanks lines before the actual configs?

Reply:
I see now that this file doesn't end with a new line, I will add that